GrandCentralBoard/Widgets/GitHub/GitHubTableDataSource.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  GitHubTableDataSource.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 21/05/16.
+//
+
+import UIKit
+
+
+class GitHubTableDataSource: NSObject, UITableViewDataSource {
```

Comment:
The protocol needs it
`public protocol UITableViewDataSource : NSObjectProtocol`